src/VideoManager/VideoReceiver/GStreamer/GStreamer.cc

```diff
@@ -182,18 +182,25 @@ GStreamer::initialize(int argc, char* argv[], int debuglevel)
     #endif
 #elif defined(Q_OS_WIN)
     QString currentDir = QCoreApplication::applicationDirPath();
-    qgcputenv("GST_PLUGIN_PATH", currentDir, "/gstreamer-plugins");
+    qgcputenv("GST_PLUGIN_SCANNER", "C:/gstreamer/1.0/msvc_x86_64", "/libexec/gstreamer-1.0/gst-plugin-scanner");
+    qgcputenv("GST_PTP_HELPER_1_0", "C:/gstreamer/1.0/msvc_x86_64", "/libexec/gstreamer-1.0/gst-ptp-helper");
+    qgcputenv("GTK_PATH", "C:/gstreamer/1.0/msvc_x86_64", "");
+    qgcputenv("GIO_EXTRA_MODULES", "C:/gstreamer/1.0/msvc_x86_64", "lib/gio/modules");
+    qgcputenv("GST_PLUGIN_SYSTEM_PATH_1_0", "C:/gstreamer/1.0/msvc_x86_64", "/lib");
+    qgcputenv("GST_PLUGIN_SYSTEM_PATH", "C:/gstreamer/1.0/msvc_x86_64", "/lib");
+    qgcputenv("GST_PLUGIN_PATH_1_0", currentDir, "");
+    qgcputenv("GST_PLUGIN_PATH", currentDir, "");
 #elif defined(Q_OS_LINUX)
-    const QString currentDir = QCoreApplication::applicationDirPath();
-    qgcputenv("GST_REGISTRY_REUSE_PLUGIN_SCANNER", "no", "");
-    qgcputenv("GST_PLUGIN_SCANNER", "/usr/lib/x86_64-linux-gnu", "/gstreamer1.0/gstreamer-1.0/gst-plugin-scanner");
-    qgcputenv("GST_PTP_HELPER_1_0", "/usr/lib/x86_64-linux-gnu", "/gstreamer1.0/gstreamer-1.0/gst-ptp-helper");
-    qgcputenv("GTK_PATH", "/usr", "");
-    qgcputenv("GIO_EXTRA_MODULES", "/usr/lib/x86_64-linux-gnu", "/gio/modules");
-    qgcputenv("GST_PLUGIN_SYSTEM_PATH_1_0", "/usr/lib/x86_64-linux-gnu", "/gstreamer-1.0");
-    qgcputenv("GST_PLUGIN_SYSTEM_PATH", "/usr/lib/x86_64-linux-gnu", "/gstreamer-1.0");
-    qgcputenv("GST_PLUGIN_PATH_1_0", currentDir, "../lib");
-    qgcputenv("GST_PLUGIN_PATH", currentDir, "../lib");
+    // const QString currentDir = QCoreApplication::applicationDirPath();
```

Comment:
Why leave in commented out?

Reply:
I was still messing with those when I had time. I'm having a hard time getting it perfect for all systems